client/src/main/java/io/pravega/client/connection/impl/ClientConnection.java

```diff
@@ -26,6 +27,9 @@
  */
 public interface ClientConnection extends AutoCloseable {
 
+
+    PravegaNodeUri NODE_LOCATION = null;
```

Comment:
I don't think this field is needed. It's fine for the interface to simply provide the method signature and leave it up to the implementations.

Reply:
Incorporated changes